src/scss/components/_scenarios.scss

```diff
@@ -0,0 +1,106 @@
+.scenarios {
+  $main: &;
+
+  display: flex;
```

Comment:
You don't need the properties `display: flex;`, `flex-direction: column;` because you are positioning the elements absolutely so flex will not take effect here.
Remove also the properties `position: relative;` and `width: 100%;` because they are not doing anything

Reply:
Done. Removed!